abel/tests/test_direct.py

```diff
@@ -8,109 +8,77 @@
 import abel
 
 def test_direct_shape():
-    if not abel.direct.cython_ext:
-        raise SkipTest
+    # if not abel.direct.cython_ext:
+    #     raise SkipTest
     n = 21
     x = np.ones((n, n))
 
     recon = abel.direct.direct_transform(x, direction='forward')
-
     assert recon.shape == (n, n) 
 
-    recon = abel.direct.direct_transform(x, direction="inverse")
-
+    recon = abel.direct.direct_transform(x, direction='inverse')
     assert recon.shape == (n, n)
 
 
 def test_direct_zeros():
     # just a sanity check
-    if not abel.direct.cython_ext:
-        raise SkipTest
```

Comment:
This was intended to skip this test if the cython implemented failed to compile. We don't want this anymore?